docs/configuration.md

```diff
@@ -27,7 +27,7 @@ This document provides a comprehensive list of all configurable properties in th
 | config.export.enabled                    | CONFIG_EXPORT_ENABLED                    | true                 | No       | -         | Enables or disables DIAL Core configuration file scheduled export functionality                                                                       |
 | config.version.target                    | CORE_CONFIG_VERSION                      | - | Yes, if ENABLE_CORE_CONFIG_VERSION_AUTO_DETECT = false (recommended to adjust for target environment)       | -         | Version of DIAL Core configuration used in file export functionality                                                                        |
 | config.version.autoDetect.enabled        | ENABLE_CORE_CONFIG_VERSION_AUTO_DETECT   | false                | Enable auto-detection of DIAL Core version                                                                                                  |
-| config.version.autoDetect.cacheExpirationMs    | CORE_VERSION_CACHE_EXPIRATION_MS    | 300000              | Cached version expiration in milliseconds for Core version auto-detection                                                                   |
+| config.version.autoDetect.schedule.delayMs        | CORE_CONFIG_VERSION_AUTO_DETECT_SCHEDULE_DELAY_MILLISECONDS   | 300000              | Interval in milliseconds between calls to the /version endpoint on core                                                                                                  |
```

Comment:
previous name i liked more. but if you want to left this one than mark PR as breaking change (ENVVAR changed)

Reply:
related to https://github.com/epam/ai-dial-admin-backend/pull/494#discussion_r2594995435.
if we will reset to null in case of ex in scheduled job than next attempt to update will be in CORE_CONFIG_VERSION_AUTO_DETECT_SCHEDULE_DELAY_MILLISECONDS. prev implementation with cache starts truing to detect version on every call if version were lost. and in this terms prev implementation looks more preferable for me (at least for usecase for temporary network glitch)